client/app/bundles/server-components/components/TogglePanel.jsx

```
@@ -0,0 +1,34 @@
+        >
+          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
+        </svg>
+      </button>
+      {isOpen && (
+        <div className="px-4 py-3 bg-white">
+          {children}
+        </div>
+      )}
+    </div>
+  );
+};
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)